db/migrate/20161210072212_add_id_and_admin_to_groups_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddIdAndAdminToGroupsUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :groups_users, :id, :primary_key
+    add_column :groups_users, :admin, :boolean, null: false, default: false
+
+    # Add a compound index to optimize queries
```

Comment:
Style: Not sure this comment is necessary, but then again, it's in a migration so it's not a huge deal.